utils/healthcheck.py

```diff
@@ -28,7 +28,8 @@ def healthcheck(name:str='',
     """
     def dec(f):
         f.name = name
-        f.remediation = remediation
+        if not hasattr(f, remediation):
```

Comment:
check_working_directory() seems to be the only one where there are multiple possible remediations, thus the decorator remediation is redundant.

this suggests that for clarity we should remove the decorator remediation and the function's remediation setting is used everywhere. do you think this is the best solution, or is there something else we can try that will maintain the clarity of the code?

Reply:
Removed decorator remediation in 58eb9560b89b07acd9cd86b5b3a400485c20150c